k8s/cse/import-corner/d2cc/data-bridge/cse-import-d2cc-automated-forced-pras-data-bridge-configmap.yaml

```diff
@@ -0,0 +1,35 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cse-import-d2cc-automated-forced-pras-data-bridge-configmap
```

Comment:
If you change name in deployment you have to change the name here too `cse-import-d2cc-automated-forced-pras-data-bridge`